### ISSUE TITLE
How can I accumute the total loss if using a closure for the optimizer?

### ISSUE THREAD

**Issue by stephane-archer** [pytorch/pytorch#13504]
I was doing `total_loss += loss.detach()` after `optimizer.step()` but now I use a closure.
How can I have access access to the loss now?

I use the `total_loss` to know when to stop the learning process.
